bin/swagger-tools

```diff
@@ -35,9 +35,13 @@ var path = require('path');
 var pkg = require('../package.json');
 var program = require('commander');
 var request = require('superagent');
-var S = require('string');
 var YAML = require('js-yaml');
 
+String.prototype.padRight = function (length) {
+  var pad = Array(256).join(' ');
```

Comment:
Using a constant of `256` here, is that intentional?  `string#padRight` doesn't do this: https://github.com/jprichardson/string.js/blob/21eb9f67a545d9320558b20876b1551e9f38e52f/lib/string.js#L259

Reply:
Should have went straight to the source in the first place. Changed:)